KabuKit/Classes/core/Swift/Screen.swift

```diff
@@ -26,9 +26,9 @@ public protocol Screen : class {
      - Attention :
      ただし、前のシーンがない場合は戻らず、何も起きない
      */
-    func leave() -> Void
```

Comment:
Thank you for your comment.
I will fix this with overloading as you recommended 👍